content/en/docs/tasks/observability/distributed-tracing/lightstep/index.md

```diff
@@ -62,31 +62,10 @@ use your own cert and your own pool's endpoint (`host:port`).
 
 Follow these steps if you're using the Public or Developer Mode Satellites, or if you're using on-premise Satellites with a TLS certificate.
 
-1.  Deploy Istio with the following configuration parameters specified:
-    - `pilot.traceSampling=100`
-    - `global.proxy.tracer="lightstep"`
-    - `meshConfig.defaultConfig.tracing.tlsSettings.mode="SIMPLE"`
-    - `meshConfig.defaultConfig.tracing.tlsSettings.caCertificates="/etc/lightstep/cacert.pem"`
-    - `global.tracer.lightstep.address="ingest.lightstep.com:443"`
-    - `global.tracer.lightstep.accessToken="<access-token>"`
-
-    You can set these parameters using the `--set key=value` syntax
-    when you run the install command. For example:
-
-    {{< text bash >}}
-    $ istioctl install \
-        --set values.pilot.traceSampling=100 \
-        --set values.global.proxy.tracer="lightstep" \
-        --set meshConfig.defaultConfig.tracing.tlsSettings.mode="SIMPLE" \
-        --set meshConfig.defaultConfig.tracing.tlsSettings.caCertificates="/etc/lightstep/cacert.pem" \
-        --set values.global.tracer.lightstep.
-        address="ingest.lightstep.com:443" \
-        --set values.global.tracer.lightstep.accessToken="<access-token>" \
-    {{< /text >}}
-
-1.  Store your satellite pool's certificate authority certificate as a secret in the default namespace.
+1.  Store your satellite pool's certificate authority certificate as a secret in the default and `istio-system` namespace.
     Download and use [this certificate](https://docs.lightstep.com/docs/instrument-with-istio-as-your-service-mesh#cacertpem-file).
     If you deploy the Bookinfo application in a different namespace, create the secret in that namespace instead.
+    The secret also needs to be created at `istio-system`, which will be used by Istio gateways.
```

Comment:
I don't know I like this last sentence since it's a repeat of being added to the first sentence. 

Options:
 - change the first sentence back, and move this sentence after it and s/at `Istio-system`/in the `istio-system` namespace.
 - combine the two sentences: maybe 
       Store your satellite pool's certificate authority certificate as a secret in the default and `istio-system` namespaces, the later for use by the Istio gateways.

Reply:
I combined it with the first sentence. Thanks! Although I am not sure whether it should be `the later` or `the latter`.

Reply:
My bad, should have been latter.